product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Notifications 
+---
+
+BigAnimal supports receiving in-app and email notifications and allows you to choose receive notifications of a selected type. 
```

Comment:
not sure we can have two verbs in sequence `choose receive`.... but maybe I am wrong

Reply:
How about this "With BigAnimal, you can opt to get specific types of notifications and receive both in-app and email notifications."